Sources/SharedServices/UserService.swift

```diff
@@ -96,6 +96,11 @@ public struct UserService {
     public func getUserWith(githubID id: String) async throws -> DynamoUser? {
         try await userRepo.getUser(github: id)
     }
+
+    /// Links an existing discord user to a github account.
+    public func linkUser(discordID: String, githubID: String) async throws {
+        try await userRepo.linkGithub(with: discordID, githubID)
```

Comment:
This whole thing seems too confusing, maybe try something like `linkGithubID(discordID: String, githubID: String)` then also change the userRepo function to something more readable like the same thing as this function.